spring-core/src/test/java/service/EventServiceTest.java

```diff
@@ -0,0 +1,79 @@
+package service;
+
+import dao.EventDao;
+import model.Event;
+import org.assertj.core.util.Lists;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.Date;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyLong;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+class EventServiceTest {
+
+    @Mock EventDao eventDao;
+    EventService eventService;
+
+    static Event testEvent;
+
+    @BeforeAll
+    static void beforeAll() {
```

Comment:
also you can setUp stub object with annotation Lombok @Data @Builder 

field will be like here 

Event.builder()
.id(1)
.title("test event")
.date(new Date)
.build();